packages/core/src/internal/observers.ts

```diff
@@ -17,7 +17,6 @@
  * limitations under the License.
  */
 
-import { observer } from '.'
```

Comment:
I'm not sure if this import had a purpose, since it doesn't seem to be used in this file. It seems like a circular import issue, since `./index.ts` has `import * as observer from './observers'`

Reply:
It's not used at all, probably some issue with the auto-complete. Thanks for catching.